src/components/form/file_picker/_file_picker.scss

```
@@ -150,16 +152,17 @@
 // Focus
 .euiFilePicker__showDrop .euiFilePicker__prompt,
 .euiFilePicker__input:focus + .euiFilePicker__prompt {
-  @include euiFormControlFocusStyle;
+  border-color: $euiColorPrimary;
 
   .euiFilePicker--compressed & {
-    @include euiFormControlFocusStyle($borderOnly: true);
+    border-color: $euiColorPrimary;
   }
```

The extra compressed modifier is no longer necessary since both rules are doing the same thing now
```suggestion
  border-color: $euiColorPrimary;
```